### ISSUE TITLE
filmicrgb rvb : possible misleading comment - clarification needed?

### ISSUE THREAD

**Issue by TurboGit** [darktable-org/darktable#10956]
**Describe the bug/issue**

In `filmicrgb rvb` the slider : `mid-tones saturation`

We have in the tooltip:

```
desaturates the output of the module
specifically at extreme luminances.
increase if shadows and/or highlights are under-saturated.
```
So the slider is about **mid-tones**, but we talk about **extreme luminances**.

Is that a copy/paste error?

Can we in any case improve the wording to something less confusing?

**Comment by gi-man** [comment]
This is similar to this issue. #10777

**Comment by TurboGit** [comment]
Indeed, but I'll close #10777 which has not much information and do a reference to another forum. That's the kind of report I generally skip as clicking to some random URL and having to read the threads there to get the information is too time consuming.

**Comment by TurboGit** [comment]
Thanks!

**Comment by TurboGit** [comment]
Merged in master and darktable-3.8.x branch.

**Comment by victoryforce** [comment]
@TurboGit This commit was merged in darktable-3.8.x branch and it amends the translatable text. So we also need the new darktable.pot in this branch to update translation for 3.8.1. Unfortunately, it is not possible to just merge the current master translation in darktable-3.8.x , as it is already contains post-3.8 changes.

**Comment by TurboGit** [comment]
This is a new string and I think it will be ok to use the master .po file in the branch. I'll do all this before creating the 3.8.1 tarball.
